src/redux/rockets/rocketsSlice.js

```diff
@@ -36,6 +36,7 @@ const rocketsSlice = createSlice({
             }
             newRocketsArr.push(newRocket);
           });
+          console.log(newRocketsArr);
```

Comment:
You can remove this line since it's better to just use console logs to debug and then remove them. 👍🏼

Reply:
Thanks!! I forgot it.